hera_mc/cm_sysutils.py

```diff
@@ -576,6 +578,10 @@ def node_info(node_num, session=None):
     hu = cm_hookup.Hookup(session)
     na_from_file = node_antennas('file', session=session)
     na_from_hookup = node_antennas(hu, session=session)
+    if node_num == 'active':
+        node_num = sorted(list(na_from_hookup))
+    elif node_num == 'all':
+        node_num = sorted(list(na_from_file))
```

Comment:
Not a big deal, but I don't think you need the `list`, I think `sorted` will take care of that.